### PR TITLE
Consistency checks for action unit tests

### PR DESCRIPTION
**Pull request by ethantkoenig**
Adds consistency checks for the `action` table to unit tests. Also fixes an inconsistent test fixture.

### CODE REVIEW

**Comment by andreynering** [comment]
LGTM

File: models/fixtures/action.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@
   act_user_name: user3
   repo_id: 3
   repo_user_name: user3
-  repo_name: repo3 # TODO old or new name?
-  is_private: false
+  repo_name: repo3
+  is_private: true
```
Comment (strk) [inline models/fixtures/action.yml:21]: Why is the is_private value changed here ? The commit log doesn't say anything about this

Comment (ethantkoenig) [reply models/fixtures/action.yml:21]: The repo with id 3 is private, so this action should also be marked as private. I'll update the PR description to include this.

**Comment by lunny** [comment]
LGTM

**Pull request merged**